tests/vmcheck/test.sh

```diff
@@ -134,16 +134,17 @@ for tf in $(find . -name 'test-*.sh' | sort); do
         break
     fi
 
+    echo "Restoring original vmcheck commit" >> ${LOG}
+    vm_cmd ostree reset vmcheck vmcheck_orig &>> ${LOG}
+    vm_cmd ostree refs vmcheck_tmp --delete # delete refs created by test
```

Comment:
Ah, I hadn't even realized we were leaking this across tests before.  That explains some issues.